src/expr.rs

```diff
@@ -292,7 +300,7 @@ ast_struct! {
     ///
     /// *This type is available if Syn is built with the `"derive"` or
     /// `"full"` feature.*
-    pub struct ExprBinary {
+    pub struct ExprBinary #full {
```

Comment:
I didn't fully understand, why `ExprBinary` is available, if feature `full` __or__ `derive` is present. All the surrounding items only are enabled on `full`. 
I (accidentally) added `#full` to keep it consistent, yet the doc-comment would be wrong. What should be done here?

Reply:
Seems that this leads to an build failure (didn't test locally...). I'll fix it by removing the `#full` again.
The question, whether it should be `derive` and `full` is still open though.